README.md

```diff
@@ -1,4 +1,54 @@
 # Hacktoberfest2022
-A hacktoberfest2022 repo made lately to accept valid open source contribution.
+
+## _A hacktoberfest2022 repo made lately to accept valid open source contribution._
+### How to Contribute to this repository
+
+1. Fork the repository (Click the Fork button in the top right of this page,
+   click your Profile Image)
+2. Clone the forked repository to your local machine.
+
+```markdown
+git clone https://github.com/your-username/Hacktoberfest2022.git
+```
+
+3. change the present working directory
+
+```markdown
+cd Hacktoberfest2022
+```
+
```

Comment:
Where is 4 ? 
You can add:
- Create new branch
- ```git checkout -b <branch-name>```

Reply:
Done